src/PrestaShopBundle/Controller/Admin/MultistoreController.php

```diff
@@ -105,4 +107,30 @@ public function header(bool $lockedToAllShopContext): Response
             'lockedToAllShopContext' => $lockedToAllShopContext,
         ]);
     }
+
+    /**
+     * @param ShopConfigurationInterface $configuration
+     * @param string $configurationKey
+     *
+     * @return Response
+     */
+    public function configurationDropdown(ShopConfigurationInterface $configuration, string $configurationKey): Response
+    {
+        $groupList = $this->entityManager->getRepository(ShopGroup::class)->findBy(['active' => true]);
+
+        foreach ($groupList as $group) {
+            foreach ($group->getShops() as $shop) {
+                $shopConstraint = new ShopConstraint(
+                    $shop->getId(),
+                    $shop->getShopGroup()->getId(),
+                    true
+                );
+                $shop->isOverridden = $configuration->has($configurationKey, $shopConstraint);
```

Comment:
This parameter is not defined in `$shop`. Could you declare it, please ?

Reply:
I don't think it would make sense to add it in the shop entity.... this value is not only dependant on the shop, a same shop can have a different value for specific configuration keys and specific contexts. 

Also I don't want to pollute this class with a parameter that I only use in this case.

Reply:
No way. If this attribute is not defined in class, it will break next level of PHPStan. And it's a bad practice to use magic attributes.

Reply:
ok, I will add a `isOverridden()` method then, with configuration class and configuration key as parameters